bool connect
float32 leftX
float32 leftY
float32 rightX
float32 rightY
float32 leftTrg
float32 rightTrg
bool A
bool B
bool X
bool Y
bool padUp
bool padDown
bool padLeft
bool padRight
bool bumperLeft
bool bumperRight
bool guide
